lms/resources/lti_launch.py

```diff
@@ -39,28 +39,6 @@ def get_or_create_course(self):
             extra=self._course_extra(),
         )
 
-    @property
```

Comment:
Can't comment on `def get_or_create_course` line but 

- Should it be a `@property def course()` I don't dislike the `get_or_create_name` but a lof of things in this resource seem to be nouns.

- Should it be `lru_cached?`